mautic/mautic
# M3: Remove deprecations from WebhookBundle

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | 
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | https://github.com/mautic/mautic/issues/7990
| BC breaks? | Yes
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
Remove deprected code from WebhookBundle

Steps to test this PR:
1. Load up [this PR](https://mautibox.com/)
2. Use form to create/edit webhook and make sure it's working as before
3. Use defined webhook and make sure that requests are filling log
4. Use webhook in campaign action

#### List backwards compatibility breaks:
1. Removed class Mautic\WebhookBundle\EventListener\WebhookSubscriberBase
2. & not used for first parameter in constructor anymore in \Mautic\WebhookBundle\Event\WebhookEvent
3. Mautic\WebhookBundle\Model\WebhookModel::webhookStart (protected property)
4. \Mautic\WebhookBundle\Form\Type\WebhookType doesn't use translator in constructor
5. Removed WebhookBundle\EventListener\WebhookModelTrait
